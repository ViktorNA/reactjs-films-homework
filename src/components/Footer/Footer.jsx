import React from 'react';
import styles from './footer.scss';


const Footer = () => (
  <footer>
    <div className={styles.films}>
                    FILMS
    </div>
    <div className={styles.copyrate}>
                    COPYRATING 2019 SOME OTHER TEXT
    </div>
  </footer>
);

export default Footer;
