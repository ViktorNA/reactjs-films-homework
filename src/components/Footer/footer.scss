.copyright {
  color: rgba(36, 43, 45, 0.4);
  font-size: 12px;
  text-align: center;
}

.logo {
  color: #09aee4;
  font-size: 25px;
  text-align: center;
  margin-top: 90px;
  margin-bottom: 40px;
}
