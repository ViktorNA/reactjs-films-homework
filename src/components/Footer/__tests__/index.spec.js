import TestRenderer from 'react-test-renderer';
import React from 'react';
import Footer from '..';

describe('Snapshot test', () => {
  it('works correctly', () => {
    const testRender = TestRenderer.create(<Footer />);
    expect(testRender.toJSON()).toMatchSnapshot();
  });
});
