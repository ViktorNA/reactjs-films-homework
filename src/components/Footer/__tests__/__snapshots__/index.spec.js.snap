// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test works correctly 1`] = `
<footer>
  <div
    className="logo"
  >
    FILMS
  </div>
  <div
    className="copyright"
  >
    COPYRATING 2019 SOME OTHER TEXT
  </div>
</footer>
`;
